performance fixes + raycast system

MarkinoTeck opened the pull request:
added raycast system
added uninstall function
more precise killing of entities
removed fletching_table and spawner support (seemed useless to me) removed armour stand system

MarkinoTeck committed:
performance fixes + raycast system

added raycast system
added uninstall function
more precise killing of entities
removed fletching_table and spawner support (seemed useless to me)
removed armour stand system

Voxxin commented on data/lock_block/tags/blocks/full_block_lock.json:
```diff
@@ -3,15 +3,13 @@
         "minecraft:composter",
         "minecraft:crafting_table",
         "minecraft:cartography_table",
-        "minecraft:fletching_table",
```
Ideally, I'd like to keep this, in case then next big Minecarft version(s) bring functionality to the Fletching table finally.

MarkinoTeck committed:
bug fixes

added fletching_table support
fixed a blocktag that was making the raycast not working
